npp.sls

```diff
@@ -70,6 +77,9 @@ npp:
 # correctly in `pkg.list_pkgs` and to allow for removal using `pkg.remove`
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   {% for version in [
+    '8.6.2',
+    '8.6.1',
+    '8.6',
```

Comment:
You don't want these here. There are still assets for them on github.

Reply:
I see, indeed don't need the x64 so can removes these.